src/index.css

```diff
@@ -924,6 +925,15 @@ hr.horizontalLine.blue {
 ::-webkit-scrollbar {
   @apply w-1 h-1;
 }
+.card-loader-height {
+  @apply min-h-[7.188rem] sm:min-h-[17rem] lg:min-h-[16rem] xl:min-h-[9.5rem] 3xl:min-h-[8.5rem];
+}
+.sessions-loader-height {
+  @apply min-h-[24rem];
```

Comment:
for a single class why did u have to create a unique class name.
Use it directly.